scripts/common.py

```diff
@@ -81,3 +82,23 @@ def inner():
             return False
 
     return call_rate_limit_aware(inner)
+
+
+def get_wrappers(smkfile):
+    wrappers = {}
+    for match in re.finditer(r"\n\s*wrapper:\s*(.+)\n", smkfile):
+        wrapper_line = match.group(1).strip().replace('"', "")
+        wrp_clean = wrapper_line.split("/")
+        if len(wrp_clean) != 4:
+            continue
+        version, *rest = wrp_clean
+        if not re.match(r"v\d+\.\d+\.\d+", version):
+            continue
+        wrapper_name = "/".join(rest)
+        wrapper_url = f"https://snakemake-wrappers.readthedocs.io/en/stable/wrappers/{"/".join(rest)}.html"
```

Comment:
This will not work for wrapper versions from before the hackathon, because the update to the wrapper catalog changed the URL hierarchy a bit (to include the fields `bio`, `phys`, etc.) in it. But I guess this is something we should actually fix in the wrapper catalog, by including redirects for the older wrapper versions somehow...